django/chat/templates/chat/components/chat_tour_steps.html

```diff
@@ -0,0 +1,502 @@
+{% load i18n %}
+
+<script>
+startTour = () => {
+  if (document.querySelectorAll("div.message-outer").length > 0) {
+    const redirect_url = "{% url 'chat:chat_with_ai' %}?start_tour=true";
```

Comment:
The tour requires a blank chat to start.
All the actual tour content is in this file (and otto_tour_steps.html).